artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/converter/AMQPMessageSupport.java

```diff
@@ -396,7 +364,8 @@ private static CoreMessage newMessage(long id, byte messageType, CoreMessageObje
    }
 
 
-   public static byte destinationType(Destination destination) {
+   // IMPORTANT-TODO: HOW TO GET THIS?
```

Comment:
I'm confused, if this is important than why commit the change before having figured that bit out, if it isn't then why leave this whole bit of commented out code here ?

Reply:
I thought we had fixed this.. I will take a look on this on.. thanks